server/controller/src/main/resources/sql/schema.sql

```diff
@@ -122,6 +122,7 @@ CREATE TABLE IF NOT EXISTS dataset_version
     version_name  varchar(255) NOT NULL,
     version_tag   varchar(255) ,
     version_meta  TEXT         NOT NULL,
+    files_uploaded  TEXT       ,
```

Comment:
is it better to use tinyint

Reply:
this field is using to record files blake2b digest